pkgs/by-name/ne/netclient/package.nix

```diff
@@ -1,34 +1,62 @@
-{ buildGoModule
+{ lib
+, buildGoModule
 , fetchFromGitHub
-, lib
-, libX11
+, installShellFiles
+, makeWrapper
+, nix-update-script
 , stdenv
-, darwin
+, systemd
+, ...
 }:
-
 buildGoModule rec {
   pname = "netclient";
-  version = "0.22.0";
 
+  passthru.updateScript = nix-update-script { extraArgs = [ "--version=branch=master" ]; };
+  version = "unstable-2024-01-22";
   src = fetchFromGitHub {
-    owner = "gravitl";
+    owner = "nazarewk";
     repo = "netclient";
-    rev = "v${version}";
-    hash = "sha256-7raWk4Y/ZrSaGKPLrrnD49aDALkZ+Nxycd+px8Eks10=";
+    rev = "1614e652824a731157b5d262899f8d4d7e024cf0";
+    hash = "sha256-Mg3QYWAyN0R2oi2VfvvGQA7UsJ1NkKz2lAq3ZafIl5k=";
   };
-
   vendorHash = "sha256-lRXZ9iSWQEKWmeQV1ei/G4+HvqhW9U8yUv1Qb/d2jvY=";
 
-  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Cocoa
-    ++ lib.optional stdenv.isLinux libX11;
-
+  subPackages = [
+    "."
+    # "gui" # should be packaged separately because it's another binary with many more dependencies
```

Comment:
Should this be left commented out here ?

Reply:
yes, in case somebody wondered why aren't we building GUI part.

I tried to quickly build it, but it didn't work at all so left it out completely.

Reply:
ok make sense